webui/react/src/components/Navigation.tsx

```diff
@@ -7,7 +7,6 @@ import usePolling from 'hooks/usePolling';
 import css from './Navigation.module.scss';
 import NavigationSideBar from './NavigationSideBar';
 import NavigationTabbar from './NavigationTabbar';
-import NavigationTopbar from './NavigationTopbar';
```

Comment:
`nit` might be worth clearing out the reference to topbarHeight in `themes.ts` if it's no longer being used